pkg/tools/gen/testdata/jsonschema/additional/input.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "https://example.com/schemas/ethernet.json",
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"socket": {
+			"type": "object",
```

Comment:
Should the object of the object be in the following form? Ref: https://kcl-lang.io/docs/tools/cli/openapi/spec#schema-index-signature
```python
schema Socket:
    HTTP: int = 80
    HTTPS: int = 443
    [...str]: int
```

Reply:
But in most cases additionalProperties has no default value, and converting it to a form like {str:int} should be more in line with the habits of the kcl.
For the form you mentioned, it should be a case with property, you can compare the following two examples.

Case 1: Object in object but no property
```
"socket": {
	"type": "object",
	"additionalProperties": {
		"type": "integer"
	},
	"default": {
		"HTTP": 80,
		"HTTPS": 443
	}
}
```
Result:
```
socket?: {str:int} = {"HTTP": 80, "HTTPS": 443}
```
Case 2: Object in object and have properties
```
"socket": {
	"type": "object",
	"properties": {
		"HTTP": {
			"type": "integer",
			"default": 80
		},
		"HTTPS": {
			"type": "integer",
			"default": 443
		}
	},
	"additionalProperties": {
		"type": "integer"
	}
}
```
Result: 
```
schema Socket:
    http?: int = 80
    https?: int = 443
    [...str]: int
```

In the second case I should add ellipses(...), I will fix it.

Reply:
> Case 1: Object in object but no property

OK, I see. For this case, should it may be the following form?

```python
schema Socket:
    [...str]: int = {
        "HTTP" = 80
        "HTTPS" = 443
    }
```

Reply:
> Case 2: Object in object and have properties

You are right! ❤️

Reply:
> OK, I see. For this case, should it may be the following form?

Makes sense, I'll make the change.

Reply:
> 

OK. it should be the following form

```python
schema Net: 
    socket?: Socket = {"HTTP": 80, "HTTPS": 443}
schema Socket:
    [...str]: int
```

The following form should emit a type mismatch error and I will enhance it.

```python
schema Socket:
    [...str]: int = {
        "HTTP" = 80
        "HTTPS" = 443
    }
```